app/src/main/java/org/sil/storyproducer/model/StoryIO.kt

```diff
@@ -49,8 +61,10 @@ fun Story.toJson(context: Context){
     }
 }
 
-fun storyFromJson(context: Context, storyTitle: String): Story?{
+fun storyFromJson(context: Context, storyTitle: DocumentFile): Story?{
     val filePath = "$PROJECT_DIR/$PROJECT_FILE"  // location of file
+    var fileContents: String? = null
+    // var fileContents: String? = null
```

Comment:
Could we talk about using DocumentFile type?
https://developer.android.com/reference/androidx/documentfile/provider/DocumentFile#summary